git.md

```diff
@@ -84,6 +84,55 @@ $ git reset --soft HEAD^
 $ git rebase --continue  # as before
 ```
 
+### Squash Commits
+
+Squash commits with `git rebase -i` (`i` for interactive), which gives more granular control than `git commit --amend`.
+
+Example:
+1. `git rebase -i HEAD~3` brings 3 commits older than `HEAD`:
```

Comment:
this subjectively speaking should probably be phrased as 3 commits from `HEAD` (including `HEAD`)